spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitProperties.java

```diff
@@ -325,6 +331,11 @@ public Template getTemplate() {
 		private String trustStore;
 
 		/**
+		 * Set the trust store type. See {@link RabbitConnectionFactoryBean#getTrustStoreType()}
```

Comment:
Could you remove the second sentence please? We don't use Javadoc links in the field javadoc for configuration properties as it messes up the auto-generated metadata used by your IDE.